src/pages/client/auth/Sign-up/index.tsx

```diff
@@ -0,0 +1,172 @@
+import { Link } from "react-router";
+import GoogleIcon from "../../../../components/icons/GoogleIcon";
+import ClientLayout from "../../../../layout/client";
+import "./styles.css"
+import { useState } from "react";
+import FormMessage from "../../../../components/form-message";
+
+
+function Signup(){
+
+    const [Fname , setFname]=useState("");
+    const [Username ,setUsername] = useState("");
+    const [email ,setemail] = useState("");
+    const [password1  ,setpassword1] = useState("");
+    const [password2 ,setpassword2] = useState("");
+    const [error,setError]=useState("");
+    const [consent,setConsent]=useState(false);
+
+
+
+
```

Comment:
Never leave spaces and empty new lines like this remove them all, just leave a new space when you want to separate sections: like separate js code with jsx component code